Marlin/src/lcd/marlinui.cpp

```diff
@@ -1348,7 +1348,7 @@ constexpr uint8_t epps = ENCODER_PULSES_PER_STEP;
   void MarlinUI::set_status(const char * const message, const bool persist) {
     if (alert_level) return;
 
-    TERN_(HOST_PROMPT_SUPPORT, host_action_notify(message));
+    TERN_(HOST_STATUS_NOTIFICATIONS, host_action_notify(message));
```

Comment:
One caveat with this concept is that the status message may be changed very often, either by G-code or by the host itself or one of its plugins. Note that the "Machine Ready" message, the current printing file's name, or some other non-null string is used as a response to "resetting" the status message.